package.json

```diff
@@ -7,22 +7,22 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "highlight.js": "^11.4.0",
+    "highlight.js": "^11.5.0",
     "javascript-time-ago": "^2.3.13",
     "jsonwebtoken": "^8.5.1",
     "marked": "^4.0.12",
-    "next": "^12.1.0",
+    "next": "^12.1.4",
```

Comment:
We're gonna want to use `v12.1.5-canary.4` here as it includes this fix https://github.com/vercel/next.js/pull/35954 for on-demand ISR

Reply:
https://github.com/leerob/on-demand-isr/commit/63b562a85f608d3bc213d2f0de375020fb3da1d3